pulsar/client.go

```diff
@@ -129,6 +129,8 @@ type ClientOptions struct {
 	// TLSMaxVersion contains the maximum TLS version that is acceptable. See tls.Config MaxVersion for more information.
 	TLSMaxVersion uint16
 
+	StdTlsConfig *tls.Config
```

Comment:
Or `TLSConfig *tls.Config`?

Reply:
ok, now I'd rather talk about the next two proposals, which one do you prefer.

Reply:
Has been changed to TLSConfig